### ISSUE TITLE
Add profile env file to live iso type too

### ISSUE THREAD

**Issue by schaefi** [SUSE/kiwi#755]
## Problem description

For all images which boots via dracut the .profile file is included except for live iso's because there no information is needed from that file to boot or customize the boot. However the .profile contains the kiwi_revision information which is useful for any image type.

Thus the file should be included for all image types

**Comment by probonopd** [comment]
Thanks @schaefi but it does not seem to work on `openSUSE-Tumbleweed-KDE-Live-x86_64-Snapshot20180606-Media.iso`:

```
linux@host:~> cat /.profile
cat: /.profile: No such file or directory

linux@host:~> cat /proc/cmdline 
(...) iso-scan/filename=/boot/iso/openSUSE-Tumbleweed-KDE-Live-x86_64-Snapshot20180606-Media.iso (...) 

```

**Comment by davidcassany** [comment]
@probonopd note that this is included since v9.15.4, currently in openSUSE Tumbleweed ships v9.15.2. v9.15.4 was just released in the development repo 5 days ago. It takes some days to reach Tumbleweed.

In addition, once included, in order to check this information you should extract it from the initrd (as @schaefi pointed [here](https://github.com/SUSE/kiwi/issues/753#issuecomment-393788421)) or use `rd.kiwi.debug` flag at boot in order to get boot logs stored in `/run/initramfs/log/boot.kiwi`, those logs include `.profile` data. `/.profile` is not included in the image root.
